### PR TITLE
openAPI (ticdc): Clean some dead codes in api package.

### PR DESCRIPTION
**Pull request by asddongmen**
### What problem does this PR solve?

Issue Number: ref #2411

### What is changed and how it works?
As title.

### Check List 

Tests 

 - No code

Code changes

Side effects

Related changes

### Release note 

```release-note
None
```

### CODE REVIEW

File: cdc/api/open_test.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Copyright 2021 PingCAP, Inc.
```
Comment (zhaoxinyu) [inline cdc/api/open_test.go:1]:
```suggestion
// Copyright 2022 PingCAP, Inc.
```

**Comment by zhaoxinyu** [comment]
LTGM

**Review by zhaoxinyu — APPROVED** [review]

File: cdc/api/owner.go [hunk 2]
```diff
@@ -79,13 +75,10 @@ type ownerAPI struct {
...
 
-// RegisterOwnerAPIRoutes registers routes for owner APIs.
-func RegisterOwnerAPIRoutes(router *gin.Engine, capture *capture.Capture) {
+// registerOwnerAPIRoutes registers routes for owner APIs.
+func registerOwnerAPIRoutes(router *gin.Engine, capture *capture.Capture) {
 	ownerAPI := ownerAPI{capture: capture}
-	router.POST("/capture/owner/resign", gin.WrapF(ownerAPI.handleResignOwner))
 	router.POST("/capture/owner/admin", gin.WrapF(ownerAPI.handleChangefeedAdmin))
-	router.POST("/capture/owner/rebalance_trigger", gin.WrapF(ownerAPI.handleRebalanceTrigger))
-	router.POST("/capture/owner/move_table", gin.WrapF(ownerAPI.handleMoveTable))
```
Comment (overvenus) [inline cdc/api/owner.go:88]: It's kind of breaking change if we remove them directly. Could you make them as alias to the new open api?

**Review by hi-rustin — COMMENTED** [review]
@asddongmen Do you have time to advance it?

**Pull request closed without merging**